README.md

```diff
@@ -1,71 +1,107 @@
-# rex
+# REX
```

Comment:
oh - a bit of solid reasoning here - rex is actually an abbreviation of Really Excellent, which, despite the typography joke in the readme, isn't really an acronym, and therefore really shouldn't be presented as REX. this might seem minor, but if rex becomes successful I'm sure it will become the subject of endless debate if I don't stop it here, so to posterity: it's rex ;-p